sway-lsp/src/test_utils.rs

```diff
@@ -1,15 +1,35 @@
+#![allow(unused)]
```

Comment:
Shouldn't this be removed / are these used anywhere to make sure things work?

Reply:
The idea is for this file to have convenience functions for accessing different examples when developing the server. These used to be placed on top of each function but after moving these over to this file I figured it was less noisy to just have this statement at the top of the file. 

Not having this in the past had led to unnecessary changes on git when testing various things and this just seemed like a simpler approach to facilitate rapid testing.